dataactvalidator/config/sqlrules/a21_appropriations.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+    row_number,
+    availability_type_code
+FROM appropriation
+WHERE submission_id = {}
+AND LOWER(COALESCE(availability_type_code,'')) NOT IN ('X','x','')
```

Comment:
Good catch, that got left in from experimenting with different forms, it's now been removed.